skia-safe/src/modules/svg.rs

```diff
@@ -208,28 +208,8 @@ fn decode_base64(value: &str) -> Vec<u8> {
         .chars()
         .filter(|&c| !is_html_space(c))
         .collect::<String>();
-    let mut input = &*without_spaces;
 
-    if input.len() % 4 == 0 {
-        if input.ends_with("==") {
-            input = &input[..input.len() - 2]
-        } else if input.ends_with('=') {
-            input = &input[..input.len() - 1]
-        }
-    }
```

Comment:
Due to the removal of padding here, `base64` crate fails to parse the string because the config `rust-skia` creates has `DecodePaddingMode::RequireCanonical` by default: https://github.com/marshallpierce/rust-base64/blob/7f81bafe0e76bcdb84f3069d81d56d77a7a24280/src/engine/general_purpose/mod.rs#L262

Reply:
An alternative solution would be to create the config with explicit `DecodePaddingMode::Indifferent`, but I don't see why padding has to be removed before passing the string to the `base64` crate.